config/deploy/production.rb

```diff
@@ -1,9 +1,15 @@
-set :deploy_host, ENV['CAPISTRANO_PRODUCTION_DEPLOY_HOST'] || ask(:deploy_host, '')
+set :deploy_host, ENV['CAPISTRANO_PRODUCTION_DEPLOY_HOST'] || ask('deploy_host (leave off stanford.edu)', '')
 set :user, ENV['CAPISTRANO_PRODUCTION_USER'] || ask(:user, '')
+set :bundle_without, %w(deployment development test).join(' ')
```

Comment:
i think this was happening automagically?  Not sure why - puppet config for bundler?